aea/connections/base.py

```diff
@@ -19,61 +19,40 @@
 # ------------------------------------------------------------------------------
 
 """The base connection package."""
-
+import asyncio
 from abc import abstractmethod, ABC
-from queue import Queue
-from typing import Optional, TYPE_CHECKING
+from asyncio import AbstractEventLoop
+from typing import TYPE_CHECKING, Optional
 
 from aea.configurations.base import ConnectionConfig
+
 if TYPE_CHECKING:
     from aea.mail.base import Envelope  # pragma: no cover
 
 
-class Channel(ABC):
```

Comment:
Can you provide some rational as to why we are dropping this?

Reply:
@marcofavorito says this is an unnecessary abstraction at the moment, hence we are dropping it for now.